project/plugins.sbt

```diff
@@ -15,7 +15,7 @@ addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
 // scripted testing
 libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
 
-libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.8.4"
+libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.9.0-RC1"
```

Comment:
first release cut out with https://github.com/scalapb/ScalaPB/pull/534

Reply:
It seems M2 is now out (and newer), perhaps switch to that. We might want to wait with merging before 0.9.0 is released, though.

Reply:
I just amended the commit with M2. Up to you to decide if we hold the merge until a final release is cut.

Reply:
I just noticed in another PR (#518) that 0.9.0-M2 does not work with jdk8 - I suspect this is unintended, so let's wait for 0.9.0